examples/siro_sandbox/README.md

```diff
@@ -45,7 +45,7 @@ habitat_baselines.rl.agent.num_pool_agents_per_type='[1,1]' \
 habitat_baselines.eval_ckpt_path_dir='data/siro_checkpoints/human-spot-GTCoord-no-config.pth' \
 habitat.simulator.habitat_sim_v0.allow_sliding=False
 ```
-Link to the checkpoint: [human-spot-GTCoord-no-config.pth](https://drive.google.com/file/d/16WAXyut6qfy2xN_TnAvgDEO1xdb59mvc/view?usp=drive_link).
+Download [human-spot-GTCoord-no-config.pth](https://drive.google.com/file/d/16WAXyut6qfy2xN_TnAvgDEO1xdb59mvc/view?usp=drive_link) checkpoint and place it under data/siro_checkpoints/human-spot-GTCoord-no-config.pth.
```

Comment:
I don't think it makes sense to link to this file. Isn't this a private GDrive link?

Reply:
My earlier advice was to remove these terminal command examples, because the public has no way to download all the necessary data yet anyway. See my earlier comment about leaving `Example Usage` essentially blank (with a todo).

Reply:
Done, looks like this now.

<img width="1052" alt="Screenshot 2023-10-12 at 11 39 21" src="https://github.com/facebookresearch/habitat-lab/assets/37709869/f7b94ab2-405f-462b-a802-eaeda0bad0e2">

Reply:
thanks for the suggestions @eundersander!